web/src/beta/features/Editor/Visualizer/index.tsx

```diff
@@ -20,7 +22,8 @@ export type Props = {
   // storytelling
   showStoryPanel?: boolean;
   selectedStory?: Story;
-  currentPageId?: string;
+  currentPage?: Page;
+  tab?: Tab;
```

Comment:
We don't need tab added. We can know if its a story based on whether `showStoryPanel` is true or not 👍🏻